sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -142,57 +145,63 @@ impl<
 		}
 	}
 
-	pub(crate) fn block_author_is_self(&self, block_author: &SB::Public) -> bool {
+	pub(crate) fn block_author_is_self(&self, block_author: &SidechainBlock::Public) -> bool {
 		self.authority.public() == *block_author
 	}
 }
 
 impl<
 		Authority,
-		PB,
-		SB,
+		ParentchainBlock,
+		SidechainBlock,
 		OCallApi,
 		StateHandler,
 		StateKey,
 		TopPoolExecutor,
 		ParentchainBlockImportTrigger,
-	> BlockImport<PB, SB>
+	> BlockImport<ParentchainBlock, SidechainBlock>
 	for BlockImporter<
 		Authority,
-		PB,
-		SB,
+		ParentchainBlock,
+		SidechainBlock,
 		OCallApi,
-		SidechainDB<SB::Block, SgxExternalities>,
+		SidechainDB<SidechainBlock::Block, SgxExternalities>,
 		StateHandler,
 		StateKey,
 		TopPoolExecutor,
 		ParentchainBlockImportTrigger,
 	> where
 	Authority: Pair,
 	Authority::Public: std::fmt::Debug,
-	PB: ParentchainBlockTrait<Hash = H256>,
-	SB: SignedBlockT<Public = Authority::Public> + 'static,
-	SB::Block: BlockT<ShardIdentifier = H256>,
+	ParentchainBlock: ParentchainBlockTrait<Hash = H256>,
+	SidechainBlock: SignedBlockTrait<Public = Authority::Public> + 'static,
+	SidechainBlock::Block: BlockTrait<ShardIdentifier = H256>,
 	OCallApi: EnclaveSidechainOCallApi + ValidateerFetch + GetStorageVerified + Send + Sync,
 	StateHandler: HandleState<StateT = SgxExternalities>,
 	StateKey: StateCrypto + Copy,
-	TopPoolExecutor: TopPoolCallOperator<PB, SB> + Send + Sync + 'static,
-	ParentchainBlockImportTrigger:
-		TriggerParentchainBlockImport<SignedParentchainBlockGeneric<PB>> + Send + Sync,
+	TopPoolExecutor: TopPoolCallOperator<ParentchainBlock, SidechainBlock> + Send + Sync + 'static,
+	ParentchainBlockImportTrigger: TriggerParentchainBlockImport<SignedParentchainBlockGeneric<ParentchainBlock>>
+		+ Send
+		+ Sync,
 {
```

Comment:
In occasions like this I get unsure, whether it would be better to use well-defined acronyms as trait parameters (more consistently used than the ones before) after all because you loose the overview:

* of opening and closing `<`
* when it extends to multiple lines.

I think it would be reasonable to use acronyms for very frequently used traits, like `ParentchainBlock`, `SidechainBlock`, `SignedSidechainBlock`

What do you think about that?

Reply:
I'm fine with that, but if we decided on something, we should put it into action immediately, otherwise it will get lost again.

Reply:
Would an alias not also solve this problem? In this case an alias for `TriggerParentchainBlockImport<SignedParentchainBlockGeneric<ParentchainBlock>>` ?

Reply:
Discussion result: We will continue using the long, but descriptive names for now.

Reply:
> TriggerParentchainBlockImport<SignedParentchainBlockGeneric<ParentchainBlock>>

The problem with that is, that `TriggerParentchainBlockImport` is a trait, and trait aliasing is not yet resolved: https://github.com/rust-lang/rust/issues/41517

I think it's safer to leave as is for now. I removed the extra "Generic" though, as Traits are now annotated with `Trait`